src/e3/main.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, name: Optional[str] = None, platform_args: bool = False):
             help="disable color and progress bars",
         )
         log_group.add_argument(
-            "--json-log",
+            "--json-logs",
```

Comment:
to keep the settings at the same place I would have just changed the default here,

default="json-logs" in os.environ.get("E3_ENABLE_FEATURE", "").split(",")

Reply:
I see but this will be enable JSON logs for normal stdout, I wanted them to be enabled only when --console-logs is used.

Reply:
Done